feat: update docs for new query server config (#328)
apache/couchdb-documentation: src/config/query-servers.rst

      for the executable ``PATH``.

    The default query server is written in :ref:`JavaScript <query-server/js>`,
    running via `Mozilla SpiderMonkey`_. It requires no special environment
    settings to enable, but is the equivalent of these two variables::

        COUCHDB_QUERY_SERVER_JAVASCRIPT="/opt/couchdb/bin/couchjs /opt/couchdb/share/server/main.js"
        COUCHDB_QUERY_SERVER_COFFEESCRIPT="/opt/couchdb/bin/couchjs /opt/couchdb/share/server/main-coffee.js"

    By default, ``couchjs`` limits the max runtime allocation to 64MiB.
    If you run into out of memory issue in your ddoc functions,
    you can adjust the memory limitation (here, increasing to 512 MiB)::

        COUCHDB_QUERY_SERVER_JAVASCRIPT="/usr/bin/couchjs -S 536870912 /usr/share/server/main.js"

    For more info about the available options, please consult ``couchjs -h``.

    .. _Mozilla SpiderMonkey: https://developer.mozilla.org/en/docs/SpiderMonkey

    .. seealso::

>         The :ref:`Mango Query Server <api/db/_find>` is a declarative language

Reviewer: The language, not the data, requires no programming :) 

```suggestion
        The :ref:`Mango Query Server <api/db/_find>` is a declarative language that requires *no programming*, 
```

>         for indexing and finding data in CouchDB that requires *no programming*.

Reviewer: ```suggestion
        and is used for indexing and finding data in CouchDB. 
```
